app/frontend/wallet/shelley-wallet.ts

```diff
@@ -160,12 +152,13 @@ const MyAddresses = ({accountIndex, cryptoProvider, gapLimit, blockchainExplorer
 }
 
 const ShelleyBlockchainExplorer = (config) => {
+  // TODO: move to separate file
   const be = BlockchainExplorer(config)
 
   const fixAddress = (address) => (isShelleyAddress(address) ? bechAddressToHex(address) : address)
   const fix = (addresses: Array<string>): Array<string> => {
     return addresses.map(fixAddress)
-  }
+  } // TODO: rename to better name than "fix"
```

Comment:
I would name it `shelleyAddressesToHex` and also I would rewrite the exported functions below that are unnecessarily verbose, I mean reriting e.g:


```
getTxHistory: (addresses) => {
      return be.getTxHistory(fix(addresses))
},
...

to
```
getTxHistory: (addresses) => be.getTxHistory(shelleyAddressesToHex(addresses)),
```